### ISSUE TITLE
Page cache clearing on-read still an issue

### ISSUE THREAD

**Issue by Didstriker** [bgmgmbh/extension-bgm_hreflang#26]
I've had a hard time debugging where this issue came from, where my typo3 would take 2 calls to cache 1 page.

And finally found it on this extension, which was already addressed before at [this PR](https://github.com/bgmgmbh/extension-bgm_hreflang/pull/20)
But, the final solution applied in [This commit](https://github.com/bgmgmbh/extension-bgm_hreflang/commit/a44fd4b3d002ad69664f10a532febb82de79312a), made it so it actually can't be deactivated; 
By [using a normal assignment](https://github.com/bgmgmbh/extension-bgm_hreflang/commit/a44fd4b3d002ad69664f10a532febb82de79312a#diff-5e15ae082e9095bfca62c8b83a7a2cdbcf8e5ac4c750ab8c832aa12cce7d7a74R27), and since this ext_localconf loads after any LocalConfiguration nor AdditionalConfiguration file, any attempt to set that value is overriden by that code.

So it would require a null coalescing assignment to be actually a feature switch; `$GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['bgm_hreflang']['feature']['clearCacheInFrontent'] ??= true;` (Btw there's a typo there)

I didn't do a PR for that because I honestly think you don't even need that code anymore for backwards compatibility, and it is somewhat obscure and hard to know about it in the first place, so I'd rather just delete all that code.

**Comment by mhuber84** [comment]
Good catch, thank you. I will publish a patched version to fix this issue.

If you set `$GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['bgm_hreflang']['feature']['clearCacheInFrontent'] = false;` in your theme extension's ext_localconf.php, it should be loaded after this extension. Perhaps this is a workaround until the new version is available.

**Comment by Didstriker** [comment]
Yeah I tried that, but my extension loads before this one too 😄. But is fine, we use a patched version of the extension until the next release comes out, thanks!
